test/liblll/EndToEndTest.cpp

```diff
@@ -386,6 +386,60 @@ BOOST_AUTO_TEST_CASE(keccak256_32bytes)
 		fromHex("b10e2d527612073b26eecdfd717e6a320cf44b4afac2b0732d9fcbe2b7fa0cf6")));
 }
 
+// This test commented until issue #2465 is resolved.
+//BOOST_AUTO_TEST_CASE(alloc_zero)
+//{
+//	char const* sourceCode = R"(
+//		(returnlll
+//			(seq
+//				(mstore 0x00 1)
+//				(alloc 0)
+//				(return 0x00 0x20)))
+//	)";
+//	compileAndRun(sourceCode);
+//	BOOST_CHECK(callFallback() == encodeArgs(u256(1)));
+//}
+
+BOOST_AUTO_TEST_CASE(alloc_size)
+{
+	char const* sourceCode = R"(
+		(returnlll
+			(return (- (msize) (alloc (calldataload 0x04)))))
+	)";
+	compileAndRun(sourceCode);
+	// Note that (msize) is always a multiple of 32.
+	// First test (alloc 0) commented until issue #2465 is resolved.
+	//BOOST_CHECK(callContractFunction("test()", 0) == encodeArgs(u256(0)));
+	BOOST_CHECK(callContractFunction("test()", 1) == encodeArgs(u256(32)));
+	BOOST_CHECK(callContractFunction("test()", 32) == encodeArgs(u256(32)));
+	BOOST_CHECK(callContractFunction("test()", 33) == encodeArgs(u256(64)));
+}
+
+BOOST_AUTO_TEST_CASE(alloc_start)
+{
+	char const* sourceCode = R"(
+		(returnlll
+			(seq
+				(mstore 0x40 0)
```

Comment:
This is only to set msize initially to something other than 0x00 so we're testing a slightly more interesting case. After this `(msize)` and hence the `(alloc 0x01)` should both evaluate to 0x60.  If we don't do this it comes out to 0x00, which is duller.